config/routes.rb

```diff
@@ -1,57 +1,11 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1, :defaults => {format: "json"} do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.